### ISSUE TITLE
Network Policy addon for container cluster is not implemented

### ISSUE THREAD

**Issue by LudovicTOURMAN** [terraform-providers/terraform-provider-google#583]
### Terraform Version
terraform : v0.10.7
terraform-provider-google : v1.0.1

### Affected Resource(s)
- google_container_cluster

### Terraform Configuration Files
```hcl
resource "google_container_cluster" "my-cluster" {
 ...
  addons_config {
    network_policy { 
      enabled = true
    }
  }
}
```

### Expected Behavior
Should enable [GPE addon for Network Policy](https://cloud.google.com/container-engine/docs/network-policy)

### Actual Behavior
Throw this error
```
1 error(s) occurred:

* module.k8s-cluster.google_container_cluster.my-cluster: addons_config.0: invalid or unknown key: network_policy
``` 

### Steps to Reproduce
1. Add block below to configuration file
2. Set `terraform apply`

**Comment by sebglon** [comment]
Has described on documentation, Network policy is on BETA on gkE kubernetes 1.7.6
https://cloud.google.com/container-engine/docs/network-policy

But this is released on Kubernetes project.

**Comment by paddycarver** [comment]
This looks like it was merged as part of #630, so I'm going to close it out.
